### ISSUE TITLE
Change maf to af_allele2 in .snp files

### ISSUE THREAD

**Issue by juhis** [FINNGEN/finemapping-pipeline#29]

**Comment by mkanai** [comment]
Good point! -- though,  FINEMAP requires MAF to be MAF, not AF... I think we can add an extra column for `.z` file and merge it back after FINEMAP as we did for `p` column.
